content/geometry/human-index.adoc

```diff
@@ -0,0 +1,30 @@
+= Human structure
+:toc:
+:toclevels: 7
+
+//include::human-structure/Floor.adoc[leveloffset=+1] Removed as stated on the Miro board
+include::human-structure/Grp_Equipment_Static_(T).adoc[leveloffset=+1]
+include::human-structure/Skeleton.adoc[leveloffset=+1]
+include::human-structure/Hip_(T).adoc[leveloffset=+2]
+include::human-structure/Lower_Spine_(T).adoc[leveloffset=+3]
+include::human-structure/Upper_Spine_(T).adoc[leveloffset=+4]
+include::human-structure/Neck_(T).adoc[leveloffset=+5]
+include::human-structure/Head_(T).adoc[leveloffset=+6]
+include::human-structure/Shoulder_R_(T).adoc[leveloffset=+4]
```

Comment:
For the following entries, in order to achieve consistency with the naming scheme of the vehicle structure; we should write "Right" and "Left" and remove the underscore before (T).